packages/core/src/domain/session/sessionState.ts

```diff
@@ -1,31 +1,58 @@
 import { isEmptyObject } from '../../tools/utils/objectUtils'
 import { objectEntries } from '../../tools/utils/polyfills'
 import { dateNow } from '../../tools/utils/timeUtils'
-import { SESSION_EXPIRATION_DELAY } from './sessionConstants'
+import { SESSION_EXPIRATION_DELAY, SESSION_TIME_OUT_DELAY } from './sessionConstants'
 
-const SESSION_ENTRY_REGEXP = /^([a-z]+)=([a-z0-9-]+)$/
+const SESSION_ENTRY_REGEXP = /^([a-zA-Z]+)=([a-z0-9-]+)$/
 const SESSION_ENTRY_SEPARATOR = '&'
 
+export const EXPIRED = '1'
+
 export interface SessionState {
   id?: string
   created?: string
   expire?: string
   lock?: string
+  isExpired?: typeof EXPIRED
 
   [key: string]: string | undefined
 }
 
-export function isSessionInExpiredState(session: SessionState) {
+export function getExpiredSessionState(): SessionState {
+  return {
+    isExpired: EXPIRED,
+  }
+}
+
+export function isSessionInNotStartedState(session: SessionState) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could be nice to add a comment to know when this state can happen (first load of the page + potential cookie deletion)